src/nIsForName.js

```diff
@@ -22,7 +22,14 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
-}
+  const result = names.filter(item => {
+    const elems = item.split(' ');
+
+    if (elems.some(x => x[0] === 'N')) {
+      return elems;
+    }
+  });
 
+  return result;
+}
```

Comment:
`return names.filter(item => item.split(' ').some(x => x[0] === 'N'))`
you can write the same much simpler